packages/hadron-build/commands/release.js

```diff
@@ -562,20 +557,18 @@ exports.run = (argv, done) => {
     task('store build configuration as json', writeConfigToJson)
   ].filter(Boolean));
 
-  return async.series(tasks, (_err) => {
-    try {
-      if (_err) {
-        return done(_err);
+  return tasks.reduce((promise, task) => {
```

Comment:
Nit: here we can probably just go with a `for..of` with `async await`? it would simplify how it reads a lot.